storybook/native/main.js

```diff
@@ -22,7 +22,9 @@ configure(() => {
 
 // add decorators
 const SpaceAround = styled.View`
-  margin: 20px;
+  padding: 20px;
```

Comment:
Can use `Space` component for this.

Reply:
Avoiding the usage of design system components here. This should be agnostic of what design system follows since it's just the story.

Reply:
My story container should not break if DS components change. Stories don't have test that can protect them from DS changes